types/Filter.d.ts

```diff
@@ -265,8 +265,8 @@ export interface VideoInputParam extends InputParam {
 	height: number
 	pixelFormat: string
 	pixelAspect: Array<number>
-	hw_device_ctx: HWDeviceContext // Optional
```

Comment:
That could be a separate PR, but I noticed the types were causing issues in my package. Looks like they are optional, so these `?` should not hurt.